tests/integration/sparse_pipeline/test_sparse_pipeline.py

```diff
@@ -66,10 +65,14 @@ def get_query_handler(self):
 
 def test_sparse_pipeline(mocker, docs_to_index):
     def validate(response):
-        print(response)
+        assert len(response.docs) == 10
```

Comment:
we should assert the length of the matches, and the type of embedding returned in the matches